ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -8304,3 +8304,79 @@ DTPM:
   - detergent_pump_run_command
   implements:
   - MONITORING
+
+DICM2X:
+  guid: "2fd7a31d-3c34-4455-a458-0e30fe207cea"
+  description: "Damper isolation control and monitoring. "
+  is_abstract: true
+  uses:
+  - supply_air_isolation_damper_closed_status_1
+  - supply_air_isolation_damper_open_status_1
+  - supply_air_isolation_damper_closed_status_2
+  - supply_air_isolation_damper_open_status_2
+  - supply_air_isolation_damper_command_1
+  - supply_air_isolation_damper_command_2
+
+PDSCV5X:
+  guid: "58a33a88-c7a9-426a-a820-8db1a99cbd0e"
+  description: "5 Pressure-dependent supply damper control for ventilation purposes (CO2 or VOC)."
+  is_abstract: true
+  uses:
+  - supply_air_damper_percentage_command_1
+  - supply_air_damper_percentage_command_2
+  - supply_air_damper_percentage_command_3
+  - supply_air_damper_percentage_command_4
+  - supply_air_damper_percentage_command_5
+  - supply_air_damper_percentage_sensor_1
+  - supply_air_damper_percentage_sensor_2
+  - supply_air_damper_percentage_sensor_3
+  - supply_air_damper_percentage_sensor_4
+  - supply_air_damper_percentage_sensor_5
+  implements:
+    - CONTROL
+
+COM4X:
+  guid: "f3c57976-61d5-41b6-8972-a35d1130a7d9"
+  description: "Carbon monoxide monitoring with 4 sensors."
+  is_abstract: true
+  implements:
+  - MONITORING
+  uses:
+  - zone_air_co_concentration_sensor_1
+  - zone_air_co_concentration_sensor_2
+  - zone_air_co_concentration_sensor_3
+  - zone_air_co_concentration_sensor_4
+
+CHWBZTC:
+  guid: "cbdecbaf-5ddb-4f77-b85f-2fbd92970a70"
+  description: "Chilled water valve binary (open/closed) control."
+  is_abstract: true
+  implements:
+  - OPERATIONAL
+  uses:
+  - chilled_water_valve_command
+  - zone_air_temperature_sensor
+  - zone_air_temperature_setpoint
+  opt_uses:
+  - failed_chilled_water_valve_alarm
+  - failed_zone_air_temperature_alarm
+  - high_zone_air_temperature_alarm
+  - low_zone_air_temperature_alarm
+
+REFSTC2X:
+  guid: "4690aaf5-e70c-43e0-aabb-38d209b8c2aa"
+  description: "Refrigerant temperature temp control for 2 circuits."
+  is_abstract: true
+  opt_uses:
+  - refrigerant_discharge_temperature_sensor_1
+  - refrigerant_discharge_temperature_sensor_2
+  - refrigerant_suction_temperature_sensor_1
+  - refrigerant_suction_temperature_sensor_2
+  uses:
+  - refrigerant_condenser_saturation_temperature_sensor_1
+  - refrigerant_condenser_saturation_temperature_sensor_2
+  - refrigerant_evaporator_saturation_temperature_sensor_1
+  - refrigerant_evaporator_saturation_temperature_sensor_2
+  - refrigerant_condenser_saturation_temperature_setpoint_1
```

Comment:
Is there only the one setpoint for both circuits? If so the enumeration is not needed

Reply:
there is only 1 and I have removed the enumeration